### ISSUE TITLE
Bounding boxes for relations stretching lon=180

### ISSUE THREAD

**Issue by plepe** [drolbr/Overpass-API#524]
Take the following object: https://www.openstreetmap.org/relation/3237099

It's a lake which crosses the longitude 180.

When using the following query:
```
[out:json];relation(3237099);out bb;
```

the resulting object stretches the whole world:
```json
 {
    "minlat": 65.8635488,
    "minlon": -180.0000000,
    "maxlat": 65.8660500,
    "maxlon": 180.0000000
  }
```

When using `out geom` though, the bounding box is calculated from its members:
```
[out:json];relation(3237099);out geom;
```
```json
{
    "minlat": 65.8635488,
    "minlon": 179.9918953,
    "maxlat": 65.8660500,
    "maxlon": -180.0000000
  }
```

As you can see, the bounds now reflect that the object stretches over lon180 (maxlon < minlon). But, maxlon=-180 is clearly wrong, it should be  -179.9872242.

**Comment by drolbr** [comment]
Thank you. This too looks like a bug, and might be related to the other one.

**Comment by drolbr** [comment]
Fixed in 486819c8be86af192fc297bb944bbb3af986eead

**Comment by plepe** [comment]
Great! Thank you :-)
